lib/Conversion/TorchToTcp/Utils.cpp

```diff
@@ -56,3 +56,23 @@ Value torch_to_tcp::broadcastShapeInLeadingDims(
   return rewriter.create<tcp::BroadcastOp>(op->getLoc(), target.getType(),
                                            input, dimSizes, axesAttr);
 }
+
+Value torch_to_tcp::broadcastInLeadingDimsToMatchShape(
+    ConversionPatternRewriter &rewriter, Value input, Value target) {
+  RankedTensorType targetType = target.getType().dyn_cast<RankedTensorType>();
+  assert(targetType);
+
+  RankedTensorType inputType = input.getType().dyn_cast<RankedTensorType>();
+  assert(inputType);
```

Comment:
doing `cast` instead of `dyn_cast` performs the `assert` automatically. I would also add a comment above this function mentioning that inputs are expected to be `RankedTensorType`

Reply:
Good point. Fixed.